zerver/tests/test_auth_backends.py

```diff
@@ -4669,6 +4669,16 @@ def test_invalid_subdomain(self) -> None:
                 dict(username=self.email, password=initial_password(self.email)),
             )
             self.assert_json_error_contains(result, "Invalid subdomain", 404)
+        do_create_realm(string_id="acme", name="acme")
```

Comment:
We should do a blank line before your new block for readability.

Reply:
DONE - moved to [PR 22060](https://github.com/zulip/zulip/pull/22060)